Misc/NEWS

```diff
@@ -10,6 +10,9 @@ What's New in Python 3.7.0 alpha 1?
 Core and Builtins
 -----------------
 
+- Issue #29779: New environment variable PYTHONHISTORY if
```

Comment:
Change `Issue #29779` to `bpo-29779`.  That's the new style of marking issue numbers.